src/main/java/com/nike/cerberus/util/AwsIamRoleArnParser.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright (c) 2017 Nike, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package com.nike.cerberus.util;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+/**
+ * Utility class for concatenating and parsing AWS IAM role ARNs.
+ */
+// TODO: remove
+public class AwsIamRoleArnParser {
+
+    public static final String AWS_IAM_ROLE_ARN_TEMPLATE = "arn:aws:iam::%s:role/%s";
+
+    private static final Pattern IAM_ROLE_ARN_PATTERN = Pattern.compile("^arn:aws:iam::(?<accountId>\\d+?):role/(?<roleName>.+)$");
+
+    public String getAccountId(String roleArn) {
+
+        Matcher iamRoleArnMatcher = IAM_ROLE_ARN_PATTERN.matcher(roleArn);
+
+        if (! iamRoleArnMatcher.find()) {throw new RuntimeException("Invalid role ARN");}
```

Comment:
Instead of RuntimeException how about define your own Exception type that extends RuntimeException?